contributors/devel/sig-testing/monitoring.md

```diff
@@ -0,0 +1,264 @@
+
+- [Monitoring Kubernetes Health](#monitoring-kubernetes-health)
+  - [Monitoring the health of Kubernetes with TestGrid](#monitoring-the-health-of-kubernetes-with-testgrid)
+    - [What dashboards should I monitor?](#what-dashboards-should-i-monitor)
+  - [What do I do when I see a TestGrid alert?](#what-do-i-do-when-i-see-a-testgrid-alert)
+    - [Communicate your findings](#communicate-your-findings)
+    - [Fill out the issue](#fill-out-the-issue)
+    - [Iterate](#iterate)
+
+
+## Monitoring the health of Kubernetes with TestGrid
+
+TestGrid is a highly-configurable, interactive dashboard for viewing your test
+results in a grid. TestGrid's back end components are open sourced and can be
+viewed here https://github.com/GoogleCloudPlatform/testgrid. The front-end code
+that renders the dashboard is not currently open sourced.
+
+The Kubernetes community has its own instance of TestGrid, https://testgrid.k8s.io/
+which we use to monitor and observe the health of the project.
+
+Each [SIG](https://github.com/kubernetes/community/blob/master/sig-list.md) has
+its own set of dashboard. Each dashboard is composed of different jobs (build,
+unit test, integration test, end-to-end (e2e) test, etc.) These views allow
+different teams to monitor and understand how their areas are doing.
+
+E2E jobs are in turn made up of test stages (e.g., bootstrapping a Kubernetes
+cluster, tearing down a Kubernetes cluster) and e2e tests are organized
+hierarchically per Component and Subcategory within that component. (e.g., the
+[Kubectl
+client](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/kubectl/kubectl.go#L229)
+component has tests that describe the expected behavior of [Kubectl
+logs](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/kubectl/kubectl.go#L1389)
+One such test is [should be able to retrieve and filter
+logs](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/kubectl/kubectl.go#L1412)).
+
+We highly encourage SIGs to periodically monitor the dashboards related to the
+sub-projects that they own. If you see that a job or test has been failing,
+please raise an issue with the corresponding SIG in either their mailing list or
+in Slack.
+
+Help maintaining tests is always needed throughout the project:
+the following contributions are always welcome
+ - [Triage Flaky Tests](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+kind%2Fflake)
+ - [Fix Flaky Test](https://github.com/kubernetes/kubernetes/issues?q=is%3Aissue+is%3Aopen+kind%2Fflake+-label%3Aneeds-triage+)
+ 
+**Note**: It is important that all SIGs periodically monitor their jobs and
+tests.
+
+Furthermore, if jobs or tests are failing or flaking, then pull requests will
+take a lot longer to be merged.
+
+### What dashboards should I monitor?
+
+This depends on what areas of Kubernetes you want to contribute to. You should
+monitor the dashboards owned by the SIG you are working with.
+
+Additionally, you should check:
```

Comment:
That is an assumption that is being made in this doc and you you are correct to say that it has not been stated. Though I think it might be better to state this up top.

Reply:
I added an Overview up top to add the "changes made to Kubernetes" context